mlir/tools/mlir-clang/Test/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
-set(PET_TOOLS_DIR ${CMAKE_CURRENT_SOURCE_DIR})
-
-set(PET_TO_MLIR_DEPENDS
-  FileCheck count not 
-  )
+set(MLIR_CLANG_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(MLIR_CLANG_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
+set(CLANG_HEADER_DIR ${LLVM_MAIN_SRC_DIR}/clang/lib/Headers)
```

Comment:
This path should be -I /some_path/mlir-clang/clang/lib/Headers and not -I /some_path/mlir-clang/**llvm**/clang/lib/Headers.

Reply:
Good catch, fixed!